### PR TITLE
add roles/groups to the vitup mock

### PR DESCRIPTION
**Pull request by ecioppettini**
This adds two tokens to the mock, and duplicates all voteplans so there is one per role (with the same external proposal ids). Also adds the parameters to the api so that they mirror the ones in vitss.

There are a quite a few places outside the mock (in iapyx) where I had to hardcode a group though, since I'm still not sure what needs to be done for each one of those (since I don't know what they are used for).

### CODE REVIEW

File: iapyx/src/bin/interactive/command.rs [hunk 1]
```diff
@@ -157,7 +158,7 @@ impl Votes {
                 let vote_plan_id_hash = Hash::from_str(&vote_plan_id)?;
                 if self.print_proposal_title {
                     let history = model.vote_plan_history(vote_plan_id_hash)?;
-                    let proposals = model.proposals()?;
+                    let proposals = model.proposals("direct")?;
```
Comment (Zeegomo) [inline iapyx/src/bin/interactive/command.rs:161]: can we add this as an additional argument to the command?

File: iapyx/src/bin/interactive/mod.rs [hunk 2]
```diff
@@ -115,8 +115,8 @@ impl CliController {
         })
     }
 
-    pub fn proposals(&self) -> Result<Vec<Proposal>, Error> {
-        self.backend_client.proposals().map_err(Into::into)
+    pub fn proposals(&self, group: &str) -> Result<Vec<FullProposalInfo>, Error> {
```
Comment (Zeegomo) [inline iapyx/src/bin/interactive/mod.rs:118]: why this change?

Comment (enzoc4) [reply iapyx/src/bin/interactive/mod.rs:118]: I'm not sure I understand the question

Comment (Zeegomo) [reply iapyx/src/bin/interactive/mod.rs:118]: Why does this return `FullProposalInfo` instead of `Proposal`?

Comment (enzoc4) [reply iapyx/src/bin/interactive/mod.rs:118]: Well, `Proposal` doesn't have the voteplan+index now, and this is used to cast votes.

**Comment by enzoc4** [comment]
I think the failures are unrelated. At least I managed to get the failing tests passing locally (although not consistently).

File: iapyx/src/load/request_generators/jormungandr/post/batch.rs [hunk 3]
```diff
@@ -32,12 +32,15 @@ impl BatchWalletRequestGen {
         batch_size: usize,
         use_v1: bool,
         update_account_before_vote: bool,
+        voting_group: &str,
     ) -> Result<Self, MultiControllerError> {
-        let proposals = multi_controller.proposals()?;
+        // FIXME: de-hardcode the group here
```
Comment (Zeegomo) [inline iapyx/src/load/request_generators/jormungandr/post/batch.rs:37]: nit: this has been de-hardcoded, I think

File: integration-tests/src/common/load.rs [hunk 4]
```diff
@@ -158,6 +158,7 @@ pub fn build_load_config(
         read_pin_from_filename: true,
         use_https: false,
         debug: false,
+        voting_group: "direct".to_string(),
```
Comment (Zeegomo) [inline integration-tests/src/common/load.rs:161]: is this still hardcoded? Not sure if this matters in tests that much though

Comment (enzoc4) [reply integration-tests/src/common/load.rs:161]: I guess so, I missed that... Although, that function has `#[allow(dead_code)]`, and I can't find any actual usage of it.

**Review by zeegomo — APPROVED** [review]
